RELEASE.md

```diff
@@ -72,7 +68,20 @@ https://pypi.org/project/ovirt-imageio/
 
 ## RHEL
 
-This parts works only inside the Red Hat network.
+This part works only inside the Red Hat network.
+
+This part also requires having rhpkg installed. To do so, make sure the Fedora
+RCM Tools repository is enabled:
+
+    $ cat /etc/yum.repos.d/rcm-tools-fedora.repo
+    [rcm-tools-fedora-rpms]
+    name=RCM Tools for Fedora $releasever (RPMs)
+    baseurl=https://download.devel.redhat.com/rel-eng/RCMTOOLS/latest-RCMTOOLS-2-F-$releasever/compose/Everything/$basearch/os/
+    enabled=1
+    gpgcheck=1
+    gpgkey=https://download.devel.redhat.com/rel-eng/RCMTOOLS/RPM-GPG-KEY-rcminternal
```

Comment:
Adding this in public document is not a good idea. It will be better to add the
way you can find the "RCM Tools" repos. I think it should be documented in a public web page in the "source" site. The page may be readable only by Red Hat people but the link can be publish in public documentaion.